ui/ChaosKit/Button.qml

```diff
@@ -7,9 +7,6 @@ import ChaosKit 1.0
 T.Button {
   id: control
 
-  property bool outlined: false
-  flat: outlined
```

Comment:
It's still possible to set the `flat` property for this component to use the "flat" style. As this style is useful, I'd rather avoid removing support for it.

The borders can go away though.

Reply:
My mistake, I didn't notice that it is used in some other places, will revert it.